source/guide_streamlit.rst

```diff
@@ -0,0 +1,178 @@
+.. highlight:: console
+
+.. author:: Torben <https://entorb.net/contact.php?origin=UberspaceLab>
+
+.. tag:: lang-python
+.. tag:: streamlit
+.. tag:: web
+.. tag:: audience-developers
+
+######
+Streamlit
+######
+
+.. tag_list::
+
+Streamlit_ is a high-level Python Web framework focussing on data visualization. This guide is based on the :lab:`Django <guide_django>` guide.
```

Comment:
I don't think it's based on the Django Guide. I gave you the Django Guide as an example for another Python based guide, no need to mention it here.

Reply:
I used that file as a starting point, so quite some of the text in this guide are copied from there, so I find it important to quote the source.